felix/fv/connectivity/conncheck.go

```diff
@@ -326,6 +326,11 @@ func (c *Checker) CheckConnectivityWithTimeoutOffset(callerSkip int, timeout tim
 				return
 			}
 		}
+
+		if c.RetriesDisabled {
+			break
+		}
+
 		completedAttempts++
```

Comment:
Nit: I'd increment the count above the `c.RetriesDisabled` check, otherwise the new log will print 0 tries.